src/Mvc/Extensions.ApiDescription.Server/src/buildMultiTargeting/Microsoft.Extensions.ApiDescription.Server.targets

```diff
@@ -17,11 +23,27 @@
       DependsOnTargets="GenerateOpenApiDocuments" />
 
   <Target Name="OpenApiGetDocuments" Returns="@(_OpenApiProjectDocuments)">
+    <ItemGroup>
+      <_Temporary Remove="@(_Temporary)" />
+      <_Temporary Include="$(TargetFrameworks)" Exclude="netcoreapp1.0;netcoreapp1.1;netcoreapp2.0" />
```

Comment:
Will we need to maintain this list? Are these just the releases that are too early for us to support?

Reply:
Yes, these are just releases the `dotnet-getdocument` tool doesn't support. The list won't change until and unless we make drop .NET Core 2.1 or 2.2 support in the tool. If we do something like that, changes here should be part of it.